### PR TITLE
Completed css-flexbox |  Mohamed Elmessiry

### PR DESCRIPTION
**Pull request by Mohamed-Elmessiry**
Screen Shot 2021-01-21 at 1 23 05 PM
Screen Shot 2021-01-21 at 1 40 08 PM
Screen Shot 2021-01-21 at 2 05 14 PM

### CODE REVIEW

File: css-flexbox/align-items.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  <style>
+    /* your styles here 👇 */
+    #card-deck{
+      display: flex;
+      min-height: 100vh;
+
+    }
+    .card-container{
+      flex-basis: calc(100% / 3);
+      align-items: center;
```
Comment (codingcodymiller) [inline css-flexbox/align-items.html:19]: The `align-items` and `justify-content` properties must be set on the flex container itself, not its children. These should be set on the `#card-deck` rule. Also, please provide a picture of your output for this file

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Mohamed! One minor change:

File: css-flexbox/align-items.html (hunk 1 above)
Comment (Mohamed-Elmessiry) [reply css-flexbox/align-items.html]: Screen Shot 2021-01-23 at 3 43 49 PM

**Review by codingcodymiller — APPROVED** [review]
Nice work!!! 🎉

**Pull request merged**